### DISCUSSION TITLE
Upgrade to 2021.4 broke the sql server metastore connection

### DISCUSSION THREAD

**Discussion by BigHoss** [datalust/seq-tickets#1430] [Help]
Hey, 

since i upgraded to 2021.4 i get the following error when i try to start seq:
´´´
A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an authority that is not trusted.)
´´´

since all our configuration is stored in there i cannot run anything.

**Comment by BigHoss** [comment]
the solution was to set *TrustServerCertificate=true;* in the connectionstring

**Reply by nblumhardt** [reply]
Thank you for the follow-up, sorry about that! We added a note to the release announcement about this, but hadn't noted it in the upgrade guide. I've added it now: https://docs.datalust.co/v2022.1/docs/upgrading#encryption-now-required-by-default-for-sql-server-metastore
